packages/manager/modules/pci/src/projects/new/payment/index.html

```diff
@@ -120,18 +120,24 @@ <h1 data-translate="pci_project_new_payment_title"></h1>
                         ></span>
                     </oui-button>
 
-                    <oui-button
-                        data-variant="primary"
-                        data-type="submit"
-                        data-variant-nav="next"
-                        data-disabled="$ctrl.isInvalidPaymentMethod()"
-                        data-ng-if="$ctrl.canProceedToCreateProject()"
-                        on-click="$ctrl.sendTrack('new_project_payment_continue')"
-                    >
-                        <span
-                            data-translate="pci_project_new_payment_btn_continue_default"
-                        ></span>
-                    </oui-button>
+                    <div class="d-flex">
+                        <oui-button
+                            data-variant="primary"
+                            data-type="submit"
+                            data-variant-nav="next"
+                            data-disabled="$ctrl.isInvalidPaymentMethod()"
```

Comment:
Shouldn't you disable the button when `$ctrl.globalLoading.finalize === true` ?

Reply:
It's part of `isInvalidPaymentMethod()` function.
so the create project button must be disabled.